cypress/e2e/api/homepage_api_tests.cy.js

```diff
@@ -0,0 +1,34 @@
+
+describe('Successfully retrieves customers (e.g., checks for the 200 status code)', () => {
+
+    it('successfully retrieves a list of customers*', () => {
+        cy.api_retrieveCustomers()
+            .then(response => {
+                expect(response.status).to.eq(200)
+            })
+
+    })
+
+    it('Paginates the customer list correctly when fetching products for page 2 with limit 10**', () => {
+        cy.api_retrieveCustomers(2, 10, 'All')
```

Comment:
💯 I remember it. I deliberately left it like this because I didn't know how to pass an object as a parameter, but I think I know how since I watched one of your playlist videos where you implemented a similar example. I'll fix this soon.

Reply:
Fixed!